<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addOrderWindow</class>
 <widget class="QDialog" name="addOrderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>372</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add Order</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="bottomMargin">
      <number>3</number>
     </property>
     <property name="horizontalSpacing">
      <number>1</number>
     </property>
     <item row="4" column="1">
      <widget class="QLineEdit" name="refferalCountEditBox"/>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="refferalLinkEditBox"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="refferalCountLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ref Count:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="refferalLinkLabel">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Ref ID:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="addOrderButton">
       <property name="text">
        <string>Add Order</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Type</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
        <item row="0" column="0">
         <widget class="QRadioButton" name="level5RadioButton">
          <property name="text">
           <string>LEVEL 5</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="level10RadioButton">
          <property name="text">
           <string>LEVEL 10</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="level30RadioButton">
          <property name="text">
           <string>LEVEL 30</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QGroupBox" name="platformGroupBox">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
       <property name="title">
        <string>Platform</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>51</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="euwRadioButton">
           <property name="text">
            <string>EUW</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="euneRadioButton">
           <property name="text">
            <string>EUNE</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
